client/src/workspace/layouts/WorkspaceBaseLayout.tsx

```diff
@@ -88,7 +89,15 @@ const WorkspaceRoot: React.FC = () => {
             />
             <AppsSideNav categories={data.categories} />
           </Sider>
-          <Outlet />
+          <ErrorBoundary
+            fallbackRender={() => (
+              <div style={{ padding: 20 }}>
```

Comment:
Unless you were told differently, I think this should be styled to be more obvious.